code/datums/pipe_datums.dm

```diff
@@ -336,6 +336,99 @@ GLOBAL_LIST_EMPTY(rpd_pipe_list)			//Some pipes we don't want to be dispensable
 	pipe_icon = "pipe-j1s"
 	rpd_dispensable = TRUE
 
+
+/// Pipe types for transit tubes.
+/datum/pipes/transit
+	pipe_type = PIPETYPE_TRANSIT
+	var/construction_type = null
```

Comment:
This variable holds a type, not an instance, so it shouldn't be typecast, right?

Reply:
As discussed on Discord, I've been getting conflicting advice about this; I think I prefer leaving off the type since `var/obj/foo/my_foo = /obj/foo/just_a_path` isn't accurate, the single-arg form of `istype` doesn't work on it, and honestly the explicit check here is more a sanity check against what TGUI says is happening here rather than a concern about an actual bug in the implementation. Maybe one day we'll get first-class support for typecasting vars to paths. I'll document it better though.